src/task_4.py

```diff
@@ -0,0 +1,14 @@
+def arg_reverse(func):
+    def input_args(*args):
+        output_list = list(args)
+        output_list.reverse()
+        return func(*output_list)
+    return input_args
+
+
+@arg_reverse
+def test(a: list, b: int):
```

Comment:
Лучше чтобы функция была универсальной и принимала любой тип (ниже описал как это сделать) и возвращает она tuple.
```suggestion
from typing import Any


@arg_reverse
def test(a: Any, b: Any) -> tuple:
```